skriptpy/core.py

```
@@ -0,0 +1,581 @@
+
+    def to_skript(self, indent=0):
+        """
+        Generates Skript code for looping over all players and executing the contained actions.
+        
+        Args:
+            indent: Number of spaces to indent the generated code.
+        
+        Returns:
+            A string containing the Skript code for the loop block with nested actions.
+        """
+        lines = [" " * indent + f'loop all players:']
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)